backend/uclapi/timetable/views.py

```diff
@@ -66,7 +66,7 @@ def get_modules_timetable_endpoint(request, *args, **kwargs):
 
     modules = module_ids.split(',')
 
-    date_filter = request.GET.get("date_filter")
+    date_filter = request.GET.get("date")
```

Comment:
The same must be done in `get_personal_timetable_endpoint()`. I'd also note that in that function they check for `KeyError`, which is unnecessary (I believe) if you do `s/request.GET[]/request.GET.get()`